### PR TITLE
Update OWNERS files

### PR DESCRIPTION
**Pull request by chaodaiG**
Update OWNERS files for some Prow components that are already actively reviewed by the GitHub handles from this PR, this will help:
- More accurate code review assignment distribution
- Get recognition from the community per https://github.com/kubernetes/community/blob/master/community-membership.md#approver

/cc @BenTheElder @cjwagner @alvaroaleman @listx

### CODE REVIEW

File: prow/cmd/sub/OWNERS [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- chaodaiG
+- cjwagner
+- listx
+- sebastienvas
```
Comment (BenTheElder) [inline prow/cmd/sub/OWNERS:7]: did we check with seb? 😅

Comment (chaodaiG) [reply prow/cmd/sub/OWNERS]: synced offline with Seb, he doesn't have the bandwidth, removed

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/hold
In case you want to get confirmation from the other folks added here. Remove /hold when ready to merge

**Pull request merged**